integreat_cms/cms/models/media/media_file.py

```diff
@@ -202,6 +204,48 @@ def thumbnail_url(self):
             f"{settings.BASE_URL}{self.thumbnail.url}?{self.last_modified.timestamp()}"
         )
 
+    @property
+    def used_in(self):
+        """
+        Return where the media file is used.
+
+        :return: name and model of the contents in which the media file is used.
+        :rtype: str
+        """
+
+        as_icon = (
+            list(self.icon_organizations.all())
+            + list(self.icon_regions.all())
+            + list(self.events.all())
+            + list(self.pages.all())
+            + list(self.pois.all())
+        )
+
+        as_content = []
+        try:
+            url = Url.objects.get(url=self.url)
+            as_content = list(url.links.all())
+        except Url.DoesNotExist:
+            as_content = []
+
+        result = []
+
+        for icon in as_icon:
+            result = result + [
+                (str(icon)).replace("&rarr;", "->")
+                + " ("
+                + icon._meta.verbose_name.title()
+                + ")"
+            ]
+        for content in as_content:
+            result = result + [
+                str(content.content_object) + " (" + str(content.content_type)[6:] + ")"
+            ]
+
+        result = list(dict.fromkeys(result))
+
+        return ", ".join(result)
```

Comment:
I think it would be cool to have the links to the respective content edit forms here.